lib/ansible/module_utils/hcloud.py

```diff
@@ -0,0 +1,60 @@
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible.module_utils.ansible_release import __version__
+from ansible.module_utils.basic import env_fallback
+
+try:
+    import hcloud
+
+    HCLOUD_AVAILABLE = True
+except ImportError:
+    HCLOUD_AVAILABLE = False
+
+
+class Hcloud(object):
+    def __init__(self, module, represent):
+        self.module = module
+        self.represent = represent
+        self.result = {"changed": False, self.represent: None}
+        if not HCLOUD_AVAILABLE:
+            module.fail_json(msg="hcloud-python must be available to use this module")
```

Comment:
there has been some improvement how to fail in this case. see https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/cloudstack.py#L60 as an example